documentation/src/docbkx/chapter12-getting-started-desktop.xml

```diff
@@ -281,6 +281,48 @@
 				job.
 			</para>
 		</tip>
+		<section id="wiring_components_output_data_streams">
+			<title>Output data streams</title>
+			<para>The "Link to..." option wires components together in the "main
+				flow". However, some components are able to produce additional
+				output
+				data streams. For example, the main feature of a Completeness
+				Analyzer is to
```

Comment:
Could you also add an image with "Completeness analyzer" output streams:complete and incomplete ?